.github/workflows/cmake.yml

```diff
@@ -0,0 +1,40 @@
+name: CMake
+
+on:
+  push:
```

Comment:
Just `on: [push, pull_request]` seem simpler and we don't really mind one extra build on tag.
Actually, does pull_request cover something that push doesn't?